### DISCUSSION TITLE
fn key shortcut?

### DISCUSSION THREAD

**Discussion by kentcdodds** [johnlindquist/kit#268] [Q&A]
I want to have a SUPER MODE keyboard shortcut which is a combination of <kbd>fn</kbd>+<kbd>ctrl</kbd>+<kbd>opt</kbd>+<kbd>cmd</kbd>+some other key

However, when I try using `fn` that doesn't work. Is there something else you use for that key?

Here's what I have:

```js
// Shortcut: fn ctrl opt cmd t
```

Hitting those keys doesn't execute the script.

**Comment by johnlindquist** [comment]
As far as I understand, the `fn` key doesn't emit keyboard events. For example, `fn+f5` only emits `f5`. So that's why you'll only see `f5` for shortcuts in apps like VS Code, etc. `fn` simply doesn't register. 

Karabiner and other lower levels tools somehow detect `fn`, but in advanced cases, I'm using Karabiner itself to launch scripts. (This was actually one of the driving reasons to create Script Kit 🤓). I'll definitely make some lessons around setting this up, but here's an example if this makes sense:

```edn
{:des "escape-mode"
 :rules [:escape-mode
         [:-t ["~/.kit/kar open-chrome-tab"]]
         [:-a ["~/.kit/kar app-launcher"]]
         [:-n ["~/.kit/kar quick-thoughts"]]
         [:-c ["~/.kit/kar auto-center-app"]]
         [:-k ["~/.kit/kar kit-discussions"]]
         ;
 ]}
```



> Related: For the basic shortcuts, you can also generate shortcuts for scripts here:

<img width="682" alt="Screen Shot 2021-05-17 at 8 17 24 AM" src="https://user-images.githubusercontent.com/36073/118503965-546c8c80-b6e8-11eb-8542-fd343bf7b57f.png">

**Reply by kentcdodds** [reply]
Oh nice. I just used that to change my hotkey to just not have the fn key 😅 Thanks!
